### DISCUSSION TITLE
Not reflecting in ui

### DISCUSSION THREAD

**Discussion by sreedharm07** [learndevopsonline/learndevopsonline#1261] [Q&A]
Hai raghu 
catalogues logs and mongodb logs all are good but catalogue not reflecting in ui 
yesterday created all the scripts and run succesfully for all 11 modules, i didnt  make any changes i just pull code from github and run ,today its not working in ui. ans logs too looks good 
i rechecked all the rout53 linked ips all are correct.
yesterday worked but today not working in ui 
can you please on this why this is happening raghu

And in rabbitmq  am getting the error of rabbitmq.service eroor.
please help on these errors
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/2fb7b0fd-14ac-44c0-b42c-36ce36383f3f)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/459194e7-a52b-4926-94c9-0a7cad446878)

**Comment by maheshbusireddy** [comment]
@sreedharm07 Everything is working good. just now i checked in ui
![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/adbaead1-0b32-4ead-bfe0-a88f9f139bb3)

**Reply by sreedharm07** [reply]
@maheshbusireddy bro  after you chek its working fine, same yesterday for payment too after you chek its working for me, and now too same am refreshing refreshing from 1 hr after you reply here i refreshed its working now 😅😅🥲
